docs/remotely-debuggable-browsers.md

```diff
@@ -1,5 +1,29 @@
 ## Remotely debuggable browsers
 
+#### Table Of Contents
+
+[Firefox](#firefox)
+  * [MacOS](#macos)
+    * [Firefox](#firefox-release)
+    * [Firefox Beta](#firefox-beta)
+    * [Firefox Developer Edition](#firefox-developer-edition-aurora)
+    * [Firefox Nightly](#firefox-nightly)
```

Comment:
Is it worth having all these sections? The only difference between any of them is the name of the app. This makes it looks like there are big differences between all of them to run this. I'd personally prefer to just have a `MacOS` section and use `Firefox.app` by default and then list the different binary names for different versions and make a note that they made need to use one of those.